.changeset/wide-views-admire.md

```
@@ -0,0 +1,17 @@
+---
+"@lynx-js/web-elements": minor
+---
+
+feat: integrate the LinearContainer Compat plugin
+
+**This is a BREAKING CHANGE**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)